src/js/app/pages/TownsList/TownsList.vue

```diff
@@ -454,6 +457,15 @@ export default {
         },
         nbPages() {
             return Math.ceil(this.filteredShantytowns.length / PER_PAGE);
+        },
+        elementsOnPage() {
+            const start = (this.currentPage - 1) * PER_PAGE;
```

Comment:
Je pense que le compte doit démarrer à 1 au lieu de 0, donc je ferais plutôt : `((this.currentPage - 1) * PER_PAGE) + 1`